README.rst

```diff
@@ -2022,7 +2022,7 @@ The example below will add two rules to the default list of PSR2 set rules:
         ->setFinder($finder)
     ;
 
-**NOTE**: ``exclude`` will work only for directories, so if you need to exclude file, try ``notPath``.
+**NOTE**: ``exclude`` will work only for directories, so if you need to exclude file, try ``notPath``. Both ``exclude`` and ``notPath`` methods accepts only relative paths to the ones defined with the ``in`` method.
```

Comment:
```suggestion
**NOTE**: ``exclude`` will work only for directories, so if you need to exclude file, try ``notPath``. Both ``exclude`` and ``notPath`` methods accept only relative paths to the ones defined with the ``in`` method.
```

Reply:
Plural - so it should be accept, should not?